### DISCUSSION TITLE
Proposal: Develop a uniform naming convention

### DISCUSSION THREAD

**Discussion by Lampese** [moonbitlang/core#143] [General]
The existing Pull requests are slightly different from the existing code in the core repository in terms of function names and test names. Is it necessary to develop a unified standard to manage?

**Comment by fantix** [comment]
As well as [doc strings](https://github.com/moonbitlang/core/pull/35#discussion_r1518641898).

I would +1 to an overall style guideline in addition to the current [CONTRIBUTING tutorial](https://github.com/moonbitlang/core/blob/main/CONTRIBUTING.md). However, I also understand that things are still moving fast at this early stage and perhaps a loose standard is preferred.

**Comment by Lampese** [comment]
> As well as [doc strings](https://github.com/moonbitlang/core/pull/35#discussion_r1518641898).
> 
> I would +1 to an overall style guideline in addition to the current [CONTRIBUTING tutorial](https://github.com/moonbitlang/core/blob/main/CONTRIBUTING.md). However, I also understand that things are still moving fast at this early stage and perhaps a loose standard is preferred.

Maybe you're right, we may have to wait a while to unify these things. This way we can maximize our productivity.

**Comment by i-shm** [comment]
If it is a simple restriction identifier style (such as `snake_case` or `CamelCase`), it should not have any impact. The current overall code style is `snake_case`.
